### PR TITLE
Added Unity Intro

### PR DESCRIPTION
**Pull request by ruiting-chen**
Added page for a basic introduction to Unity
Information on the page includes:
- Download and install Unity Hub and Unity Editor, Licensing
- Create a new Unity project
- Unity user interface

@arielchen07 please review

### CODE REVIEW

**Comment by arielchen07** [comment]
Hi! This is great introduction to Unity in general! 
I really like picture for explaining the interface. It's really intuitive and easy for beginners to understand!
Except the suggestion to bold some keywords, i would also recommend to add a section at the end to briefly talk about how to start making the game or just link to some videos. By doing this, the users will know what to do after they finish setting up and get familiared with Unity.

**Comment by arielchen07** [comment]
I just read your comments on my Pull Request and yes, I would also recommend you adding a table of contents at the start of the page like i did!

**Comment by ruiting-chen** [comment]
I bolded keywords in my doc and also added a table of contents section.

I think it's a good idea to add a section to talk about how to start making games at the end, to transition readers to start making something useful with Unity. However I think it would be kind of broad to talk about how to start making games in Unity in general. And there will be a lot stuff to write about which will make this doc very long. So I think I will provide links to tutorial video series that demonstrates how to make simple 2D and 3D games in Unity.

**Comment by arielchen07** [comment]
Great! I think referring the readers to some videos is a great idea here! We can just keep this page clean and only focus on some basics.

**Comment by arielchen07** [comment]
I've reread your page after you modified and I left some comments on some other parts. Could you take a look and let me know what you think

**Comment by ruiting-chen** [comment]
I added a section for how to get started with making your first Unity game and provided video links for making 2D and 3D games. 
I also changed the title of the page to be more precise. Please review the changes and let me know your thoughts

**Comment by arielchen07** [comment]
I notice that you cannot see my newly-added comments, so I attached the screenshots here.
Screenshot 2023-11-26 at 7 30 57 PM
Screenshot 2023-11-26 at 7 31 04 PM
Screenshot 2023-11-26 at 7 31 11 PM

I reviewed your new changes, it seems that you fixed the first and second comment that I mentioned, which is great! I also like your separate section for how to get started with making your first Unity game. It makes the whole page more structural.

By saying these, could also take a look at my comments in the third screenshot?

**Comment by ruiting-chen** [comment]
ohh I see. Yeah I agree with you that the links in reference are a bit messy, I will fix that

**Comment by ruiting-chen** [comment]
I updated the reference links. Please have a look and let me know how they look now

**Review by arielchen07 — COMMENTED** [review]
Great! It looks much organized now!

File: Topics/Tech_Stacks/Unity_Intro.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#### Download Unity Hub
+Unity Hub is a management tool that can be used to organize Unity projects, install different versions of Unity Editor, manage to licenses and access the Unity Asset Store efficiently. It can be used with the Unity Editor version 5.6 or higher.
+
+To get started, download Unity Hub from the [official Unity website](https://unity.com/download). 
+Choose the correct installer of Unity Hub for your operating system (Mac, Windows, or Linux).
+Open the installer file and follow the instructions in the Unity Hub setup window to install Unity Hub on your machine.
+
+#### License Activation:
+An activated license is needed to use Unity. When you sign in to the Unity Hub for the first time, you will be asked to add an active license. 
+At the top of Unity Hub, click "Manage licenses". If you do not see this, you can manage licenses by clicking on the preferences (or settings) icon on the top of the left tab bar beside your account icon > then select the license tab in the window that pops up.
```
[the remark this answers is not in the document]
Comment (ruiting-chen) [reply Topics/Tech_Stacks/Unity_Intro.md]: updated it

**Comment by arielchen07** [comment]
One last thing before merging this PR, do we need to add the name of this page to the readme file on the wiki?

File: Topics/Tech_Stacks/Unity_Intro.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- [Unity Manual](https://docs.unity3d.com/Manual/index.html): The official Unity website that provides information to different features and aspects that you can learn about in Unity. Note that the manual is slightly different for different versions of Unity.
+
+## Reference
+https://support.unity.com/hc/en-us/articles/360061586571-What-is-the-Unity-Hub-#:~:text=The%20Unity%20Hub%20is%20a,and%20installing%20add%2Don%20components. (What is the Unity Hub?)
+
+https://subscription.packtpub.com/book/game-development/9781801078078/2/ch02lvl1sec04/getting-started-with-the-unity-editor (Getting started with the Unity Editor)
+
+https://docs.unity3d.com/Manual/VisualStudioIntegration.html#:~:text=Unity%20automatically%20uses%20Visual%20Studio,into%20an%20existing%20Unity%20installation. (Unity Manual)
+
+https://getalow.com/unity-engine/introduction-to-unity-editor-and-unity-interface/16 (Introduction to Unity Editor and Unity Interface)
```
[the remark this answers is not in the document]
Comment (ruiting-chen) [reply Topics/Tech_Stacks/Unity_Intro.md]: I updated the format

**Comment by arielchen07** [comment]
I think it's in the FAQ
> What are some of the best practices to keep in mind for submission?
i. Give relevant names to your commit messages
ii. Summarize your contributions in the comment section of your PR.
iii. Document your approach wherever relevant. This ensures the
ease of use and understanding of your contribution for other
students; for example, if you request changes to someone else’s
PR- clearly explain your thought process. Update the readme files
on the wiki wherever appropriate.

**Comment by ruiting-chen** [comment]
I added the page name in readme and Tech_Stacks.md

**Comment by ruiting-chen** [comment]
ok! i will merge the pull request now

**Pull request merged**